.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,53 @@
+name: Prepare release
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  prepare_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set RELEASE_VERSION
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Prepare release
```

Comment:
This should be removed; can't have a step with a `uses` or a `run`.

Reply:
I believe you meant to refer to the name tag. It has been removed.